src/bin/pg_autoctl/keeper_pg_init.c

```diff
@@ -547,13 +547,23 @@ wait_until_primary_is_ready(Keeper *keeper,
 			Monitor *monitor = &(keeper->monitor);
 			KeeperStateData *keeperState = &(keeper->state);
 			int timeoutMs = PG_AUTOCTL_KEEPER_SLEEP_TIME * 1000;
+			char *emptyChannelList[] = { NULL };
 
+			(void) pgsql_listen(&(monitor->notificationClient), emptyChannelList);
```

Comment:
By the way, how does it work now? We are listening to no channel at all, how do we expect to get a notification that some state change happened in our formation and group, or even for our own node?